build.gradle.kts

```diff
@@ -72,15 +72,18 @@ plugins {
 }
 
 buildscript {
-    repositories {
-        gradlePluginPortal()
-        google()
-    }
-    dependencies {
-        classpath("gradle.plugin.com.github.johnrengelman:shadow:8.0.0")
+  repositories {
+    maven {
+      url = uri("https://plugins.gradle.org/m2/")
     }
```

Comment:
Isn't this the same as `gradlePluginPortal`?

Reply:
This is how it was written at https://plugins.gradle.org/plugin/com.github.johnrengelman.shadow